project/dexter-vs/Dexter/Common/Core/Utils/DexterConstants.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dexter.Common.Utils
+{
+    static public class DexterConstants
```

Comment:
The variables 'FALSE' is used to implement IVsSolutionEvents.
I added its to deal with old type 'Microsoft.VisualStudio.OLE.Interop.BOOL'.